README.adoc

```diff
@@ -90,14 +90,31 @@ Build the example located in the _upstream-1_ folder:
     git commit -a -m"Pages update"
     git push origin +gh-pages
     
-=== Creating References to Anchors
+=== Creating Anchors and xrefs
 
+==== Dynamic Anchor and Dynamic xref
 An anchor (for example: [[Testing the App]]) cannot appear more than once in an *.adoc* file.
 As Modular Doc files are regularly reused in the same *.adoc* file, this results in a build error.
 See link:http://asciidoctor.org/docs/user-manual/#include-multiple[Including Multiple Anchors] to negate this issue.
 
 To best understand the information above, see the example located in the repo.
 
+==== Dynamic Anchors and Hard-Coded xref
```

Comment:
This week I encountered a new issue re the repo structure and xrefs. My solution is to hard code xref when faced with this situation. I wrote this up during the week and added it today.

It's in the readme because if other people faced the situation I encountered (which they will), I'll point them to this text.